tests/test_image_to_text_example.py

```diff
@@ -21,8 +21,8 @@
             ("llava-hf/llava-v1.6-vicuna-13b-hf", 1, 23.527610042925),
             ("google/paligemma-3b-mix-224", 1, 132.8949150246155),
             ("HuggingFaceM4/idefics2-8b", 1, 21.89944593215077),
-            ("meta-llama/Llama-3.2-11B-Vision-Instruct", 1, 20.407843538649303),
-            ("tiiuae/falcon-11B-vlm", 1, 27.0566558689559327),
+            ("meta-llama/Llama-3.2-11B-Vision-Instruct", 1, 18.974541922240313),
```

Comment:
Hi @schoi-habana , can you please explain why the performance drop down here by using `--sdp_on_bf16`, according to the [doc](https://docs.habana.ai/en/latest/Release_Notes/GAUDI_Release_Notes.html#:~:text=torch._C._set_math_sdp_allow_fp16_bf16_reduction(True)%20to%20enable%20torch.nn.functional.scaled_dot_product_attention%20to%20operate%20on%20BF16.%20Without%20this%20setting%2C%20the%20inputs%20will%20always%20be%20cast%20to%20FP32.) , it should benefit the performance, right?